ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -3,20 +3,52 @@
 {% block content %}
 <h1>Speaking Practice View</h1>
 
+<p><span id="sentence_text" ></span></p>
```

Comment:
Standard convention for html id names is dashes, not underscores (i.e. "sentence-text" instead of "sentence_text").

Reply:
This goes for all id's